css/components/nav.css

```diff
@@ -0,0 +1,24 @@
+.nav__list {
+  list-style-type: none;
+  padding: 10px 10px;
```

Comment:
shortform
padding: 10px;

Reply:
the order ist "top" "left" "bottom" "right" - I just wanted top & bottom - when 10px also works fine thanks